### PR TITLE
Disabled users from entering the same email id

### PR DESCRIPTION
**Pull request by mish24**
Fix for issue #52. 
However I'm still not sure how to notify the users regarding the issue. Shall it be done by form or templates? As of now this PR doesn't allow it and keeps the user on the page.

### CODE REVIEW

**Comment by mish24** [comment]
I'll create a separate PR if required. This addresses two small issues in one.

File: oshc/oshc/settings.py [hunk 1]
```diff
@@ -95,6 +95,7 @@
 LOGIN_REDIRECT_URL = 'home'
 # and are trying to access pages requiring authentication
 LOGIN_URL = '/accounts/login/'
+LOGOUT_REDIRECT_URL = 'home'
```
Comment (amarlearning) [inline oshc/oshc/settings.py:98]: Extraneous change.

**Review by amarlearning — CHANGES_REQUESTED** [review]
You didn't add the error message. some kind of error message should be shown to the user.

File: oshc/oshc/settings.py (hunk 1 above)
Comment (mish24) [reply oshc/oshc/settings.py:98]: Okay, I'll send a different PR for this issue!

**Comment by tapaswenipathak** [comment]
@mish24 Are you still working on this pull request?

**Comment by amarlearning** [comment]
@mish24 Closing due to inactivity (we try to keep the PR list clean). Feel free to reopen when you have new changes to push :)

**Pull request closed without merging**